makefile

```diff
@@ -30,7 +30,8 @@ test:
 .PHONY: benchmark
 benchmark:
 	@mkdir -p $(buildDir)
-	go test $(testArgs) -bench=$(benchPattern) $(if $(RUN_TEST),, -run=^^$$) | tee $(buildDir)/bench.sink.out
+	go test $(testArgs) -timeout 45m -bench=$(benchPattern) $(if $(RUN_TEST),, -run=^^$$) | tee $(buildDir)/bench.sink.out
```

Comment:
I'd put this on line 7 rather than here.

Reply:
I avoided this because `testArgs` is used in more than just the `benchmark` target, and is used in the other targets. This would cause even the non-benchmark tasks to timeout after 45 minutes instead of the default of 10. This doesn't seem desirable since the normal tests should definitely finish in under 10 minutes.

We could always add a `benchArgs` or something, specifically for benchmarks, but it would only ever be used once in the `benchmark` target.